src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return {
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
+    ownerCount: data.OwnerCount,
+    previousTransactionID: data.PreviousTxnID,
+    previousTransactionLedgerVersion: data.PreviousTxnLgrSeq
```

Comment:
Well PreviousTxnID is still the account root, it's used for something else than transaction chaining. It shows the last transaction that affected the account. 

PreviousTxnID as a txn field was removed though.